packages/common/src/scopeSupportFacets/r.ts

```diff
@@ -0,0 +1,189 @@
+import type { LanguageScopeSupportFacetMap } from "./scopeSupportFacets.types";
+import { ScopeSupportFacetLevel } from "./scopeSupportFacets.types";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const { supported, unsupported, notApplicable } = ScopeSupportFacetLevel;
+
+export const rScopeSupport: LanguageScopeSupportFacetMap = {
+  "comment.line": supported,
+  "ifStatement": supported,
+  "argument.actual": supported,
+  "argumentList.formal": supported,
+  "condition.if": supported,
+  "condition.while": unsupported,
+
+  "namedFunction": supported,
+  "anonymousFunction": supported,
+  "functionName": supported,
+  "functionCall": supported,
+  "functionCallee": supported,
+
+  "name.assignment": supported,
+  "name.assignment.pattern": notApplicable,
+  "name.variable": supported,
+  "value.assignment": supported,
+  "value.variable": supported,
+  "value.variable.pattern": notApplicable,
+
+  // "interior.function": unsupported,
```

Comment:
Before this is ready to merge I would like all commented facets like this to be removed